Test/git-issues/git-issue-859.dfy

```diff
@@ -0,0 +1,25 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %dafny /noVerify /compile:4 /compileTarget:cs "%s" >> "%t"
+// RUN: %dafny /noVerify /compile:4 /compileTarget:js "%s" >> "%t"
+// RUN: %dafny /noVerify /compile:4 /compileTarget:go "%s" >> "%t"
+// RUN: %dafny /noVerify /compile:4 /compileTarget:java "%s" >> "%t"
+// RUN: %diff "%s.expect" "%t"
+
+datatype FailureCompatible = Make {
+  predicate method IsFailure() { true }
+  function method PropagateFailure(): int { 12 }
+  method Extract() returns (r: real) { }
```

Comment:
This tests that it's okay for the members to be non-ghost. These would have passed even without fixing #859. I suggest that these 3 lines be changed to use ghost members, since that would test that we only allow non-ghost members.

Reply:
Forgot to commit git-issue-859a.dfy that tests for the error cases